content/en/blog/_posts/2023-10-31-Gateway-API-GA.md

```diff
@@ -0,0 +1,152 @@
+well defined and sufficiently stable to graduate to GA. That led to a variety of
+spec clarifications, as well as some improvements to status to improve the
+overall UX when interacting with Gateway API.
+
+### Experimental channel
+Most of the changes included in this release were limited to the experimental
+channel. These include HTTPRoute timeouts, TLS config from Gateways to backends,
+WebSocket support, Gateway infrastructure labels, and more. Stay tuned for a
+follow up blog post that will cover each of these new features in detail.
+
+### Everything else
+For a full list of the changes included in this release, please refer to the
+[v1.0.0 release
+notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.1.0).
+
+## How we got here
+
+The idea of Gateway API was initially [proposed 4 years ago at KubeCon San
+Diego](https://youtu.be/Ne9UJL6irXY?si=wgtC9w8PMB5ZHil2) as the next generation
+of Ingress API. Since then, an incredible community has formed to develop what
+has likely become the most collaborative API in Kubernetes history. Over 170
+people have contributed to this API so far, and that number continues to grow.
+
+A special thank you to the 20+ [community members who agreed to take on an
+official role in the
+project](https://github.com/kubernetes-sigs/gateway-api/blob/main/OWNERS_ALIASES),
+providing some time for reviews and sharing the load of maintaining the project!
+
+We especially want to highlight the emeritus maintainers that played a pivotal
+role in the early development of this project:
+
+* [Bowei Du](https://github.com/bowei)
+* [Daneyon Hansen](https://github.com/danehans)
+* [Harry Bagdi](https://github.com/hbagdi)
+
+## Try it out
+
+Unlike other Kubernetes APIs, you don't need to upgrade to the latest version of
+Kubernetes to get the latest version of Gateway API. As long as you're running
+one of the 5 most recent minor versions of Kubernetes (1.24+), you'll be able to
+get up and running with the latest version of Gateway API.
+
+To try out the API, follow our [Getting Started
+guide](https://gateway-api.sigs.k8s.io/guides/).
+
+## What's next
+
+This release is just the beginning of a much larger journey for Gateway API, and
+there are still plenty of new features and new ideas in flight for future
+releases of the API.
+
+One of our key goals going forward is to work to stabilize and graduate other
+experimental features of the API. These include [support for service
+mesh](https://gateway-api.sigs.k8s.io/concepts/gamma/), additional route types
+([GRPCRoute][https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.GRPCRoute],
+[TCPRoute][https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.TCPRoute],
+[TLSRoute][https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.TLSRoute],
+[UDPRoute][https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.UDPRoute]),
```

Comment:
These have to use the `[label](link)` format, not the `[label][internal-link]` one.

Reply:
Fixed, thanks!